transactions/src/main/resources/org/jboss/as/txn/subsystem/LocalDescriptions.properties

```diff
@@ -9,14 +9,14 @@ transactions.remove=Removes the transaction subsystem.
 
 transactions.configuration=Various environment configuration. It could refer to core, recovery, coordinator environment, to object store configs and WildFly Transaciton Client.
 
-transactions.node-identifier=Used to set the node identifier. Each Xid that the transaction manager creates will have this identifier encoded within it and ensures the transaction manager will only recover branches which match the specified identifier. It is imperative that this identifier is unique between application server instances which share either an object store or access common resource managers.
-transactions.process-id-uuid=Indicates a strategy used for obtaining a unique identifier needed for creation of transaction instances. When true then the transaction manager generates the transaction id based on process id (PID). When set to false then the transaction manager generates the transaction id based on the referenced socket binding, i.e. the attribute 'process-id-socket-binding' is required.
+transactions.node-identifier=Used to set the node identifier. Each Xid that the transaction manager creates will have this identifier encoded within it and ensures the transaction manager will only recover branches that match the specified identifier. It is imperative that this identifier is unique between application server instances which share either an object store or access common resource managers.
+transactions.process-id-uuid=Indicates a strategy used for obtaining a unique identifier needed for the creation of transaction instances. When true, the transaction manager generates the transaction id based on the process id (PID). When set to false then the transaction manager generates the transaction id based on the referenced socket binding, i.e. the attribute 'process-id-socket-binding' is required.
```

Comment:
This is incorrect, we have other strategies such as `process-id-uuid`, for example.

Reply:
Ah wait, I see that WildFly only supports two options whereas narayana supports quite a few: ExecProcessId, FileProcessId, ManualProcessId, SocketProcessId, UuidProcessId and MBeanProcessId so I will resolve this comment.